synthtool/gcp/templates/node_library/.kokoro/trampoline.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
To make this PR easier to rollback, please don't delete this file yet.  Please add a comment describing that it's obsolete and add a TODO() to delete it.